### DISCUSSION TITLE
WPCrudRest-v1.0.0

### DISCUSSION THREAD

**Discussion by asadidebuger** [asadidebuger/WPCrudRest#2] [Polls]
This is the first successfully tested release

<hr /><em>This discussion was created from the release <a href='https://github.com/asadidebuger/WPCrudRest/releases/tag/WPCrudRest-1.0.0'>WPCrudRest-v1.0.0</a>.</em>

**Comment by MeysamRahpeyma** [comment]
سلام. وقت بخیر. ببخشید من هر کاری میکنم نمیتونم از این پلاگین استفاده کنم برای وردپرس. لطف میکنید نسخه وردپرس رو برای دانلود قرار بدید.

**Comment by asadidebuger** [comment]
سلام. این پلاگین روی ایکلیپس نصب میشه. پلاگین وردپرس نیست
